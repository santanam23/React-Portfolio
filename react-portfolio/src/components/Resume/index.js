import React from 'react';

function Resume() {
    return (
    <section>
    <div className="Resume" id="resume">
          <h2>Resume</h2>
          <p>
            My resume will be shared as a link to my LinkedIn acccount.
          </p>
          <p><a href="https://www.linkedin.com/in/miriam-santana-126aa6105/">LinkedIn Resume</a></p>
    </div>
    </section>
    );
}

export default Resume;