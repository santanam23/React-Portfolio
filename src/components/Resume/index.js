import React from 'react';

function Resume() {
  const first = ['HTML', 'CSS', 'Javascript', 'responsive design', 'react']
  const firstList = first.map(first => <ul>{first}</ul>)
  const second = ['APIs', 'Node', 'Express', 'MySQL, Sequeulze', 'MongoDB', 'Rest']
  const secondList = second.map(second => <ul>{second}</ul>)
    return (
    <section>
    <div>
      <h1 className="Resume" id="resume">Front-End Proficiency</h1>
    {firstList}
    </div>
    <div>
      <h1 className="Resume" id="resume">Back-End Proficiency</h1>
    {secondList}
    </div>
    </section>
    );
}

export default Resume;