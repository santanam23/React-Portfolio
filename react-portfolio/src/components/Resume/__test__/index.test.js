import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Resume from '..';

afterEach(cleanup)

describe('Resume component', () => {
    // First Test
    it('renders', () => {
      render(<Resume />);
    });
    it('renders', () => {
      const { asFragment } = render(<Resume />)
      expect(asFragment()).toMatchSnapshot()
    })

});