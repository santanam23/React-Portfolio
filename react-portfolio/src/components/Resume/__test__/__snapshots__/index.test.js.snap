// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component renders 1`] = `
<DocumentFragment>
  <section>
    <div
      class="Resume"
      id="resume"
    >
      <h2>
        Resume
      </h2>
      <p>
        My resume will be shared as a link to my LinkedIn acccount.
      </p>
      <p>
        <a
          href="https://www.linkedin.com/in/miriam-santana-126aa6105/"
        >
          LinkedIn Resume
        </a>
      </p>
    </div>
  </section>
</DocumentFragment>
`;
